locale/en/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -303,6 +306,56 @@ situations because **it allows you to "starve" your I/O by making
 recursive `process.nextTick()` calls**, which prevents the event loop
 from reaching the **poll** phase.
 
+#### Deduplication
+
+For the `timers` and `check` phases, there is a *single transition 
+between C to JS for multiple immediates and timers*. This deduplication
```

Comment:
Nit: No italics here. We overuse italics and bold for emphasis. Just don't do it. 😄